fsdriver/driver.go

```diff
@@ -35,19 +52,58 @@ type Driver interface {
 	// NOTE(stevvooe): We may want to actually include the path manipulation
 	// functions here, as well. They have been listed below to make the
 	// discovery process easier.
-
-	// Join(path ...string) string
-	// IsAbs(string) bool
-	// Abs(string) (string, error)
-	// Rel(base, target string) (string, error)
+	Join(pathName ...string) string
+	IsAbs(pathName string) bool
+	Rel(base, target string) (string, error)
+	Base(pathName string) string
+	Dir(pathName string) string
+	Clean(pathName string) string
+	Split(pathName string) (dir, file string)
+	Separator() byte
```

Comment:
There isn't an exported type. It's a constant. filepath.Separator == os.PathSeparator == '/' or '\\', so it's a byte currently. I am considering just to return a string for convenience since there is bunch of code that does string(filepath.Separator) in docker.